apiserver/facades/client/client/filtering.go

```diff
@@ -282,8 +282,12 @@ func buildUnitMatcherShims(u *state.Unit, patterns []string) []closurePredicate
 
 func matchPortRanges(patterns []string, portRanges ...network.PortRange) (bool, bool, error) {
 	for _, p := range portRanges {
+		splitPortStr := strings.Split(p.String(), "/")
+		pNum := splitPortStr[0]
```

Comment:
Might be nicer to use [`strings.Cut`](https://pkg.go.dev/strings#Cut) instead
```go
pNum, _, _ := strings.Cut(p.String(), "/")
```

Reply:
Thx, fixed